website/templates/base.mako

```diff
@@ -219,8 +219,8 @@
 <%def name="content_wrap()">
     <div class="watermarked">
         <div class="container ${self.container_class()}">
-
-            % if status:
+            % if status and not node:
```

Comment:
My previous comment is still unresolved: Is `node` guaranteed to be in the mako context for *all* templates that inherit from `base.mako`? If so, how?